test.rb

```diff
@@ -21,12 +21,16 @@ def condition
   end
 
   def bang
-    ! false
+    !! false
```

Comment:
Avoid the use of double negation (`!!`).<br>Do not leave space between `!` and its argument.